library/centos

```diff
@@ -28,6 +28,11 @@ i386-GitFetch: refs/heads/CentOS-6i386
 i386-GitCommit: 53cd22704a89c6ed59d43e2e70e63316026af4f7
 Architectures: amd64, i386
 
+Tags: centos7.7.1908, 7.7.1908
+GitFetch: refs/heads/CentOS-7.7.1908
```

Comment:
> couldn't find remote ref "refs/heads/CentOS-7.7.1908"

Looks like it is supposed to be `CentOS-7.7.1908-x86_64`

Reply:
crap, yeah. good catch thank you. Do you want to change it or should I re-submit?

Reply:
```suggestion
GitFetch: refs/heads/CentOS-7.7.1908-x86_64
```

I'll get it; tis a quick fix.

Reply:
Well, apparently I can't commit my own suggestion or something. I'll just do it via git cli.